src/include/sof/math/exp_fcn_hifi4.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+#ifndef __SOF_MATH_EXPONENTIAL_HIFI4_H__
+#define __SOF_MATH_EXPONENTIAL_HIFI4_H__
+
+#include <stdint.h>
+#include <stddef.h>
+
+#undef EXPONENTIAL_GENERIC
+#undef EXPONENTIAL_HIFI4
+#if defined(__XCC__)
+
+#include <xtensa/config/core-isa.h>
+#if XCHAL_HAVE_HIFI4 == 1
+ #define EXPONENTIAL_HIFI4 1
+#else
+ #define EXPONENTIAL_GENERIC 1
```

Comment:
This looks more or less like a duplicate of `exp_fcn_hifi.h`. Why not just include that file?